sys/linux/sys.txt

```diff
@@ -251,6 +251,8 @@ clone_args {
 	stack		ptr64[out, array[int8]]
 	stack_size	bytesize[stack, int64]
 	tls		ptr64[out, array[int8]]
+	set_tid		ptr64[in, array[pid]]
+	set_tid_size    bytesize[set_tid, int64]
```

Comment:
This is number of elements rather than size in bytes, right? If so, we need `len` instead of `bytesize` here.

Reply:
done